Modules/_testcapi/util.h

```diff
@@ -23,3 +23,5 @@
         assert(!PyErr_Occurred());          \
         return PyLong_FromSsize_t(_ret);    \
     } while (0)
+
+#define UNINITIALIZED_PTR ((void *)"uninitialized")
```

Comment:
In my previous tests for PyDict API, that I removed, for arguments set by ``&arg``, I initalized them with ``= Py_True``. But yeah, any value different than NULL is fine.

Using this macro is not obvious, can you please add a comment to explain why it's needed? Just explain how it should be used.

Reply:
I extracted this part of the PR into separate PR #109014, because I am going to backport it.